input/blog/2020-12-23-cake-vs-extension-release-0.3.0.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cake Visual Studio Extension Release 0.3.0
+category: Release Notes
+author: gep13
+---
+
+The Cake Team is very happy to announce the 0.3.0 release of the [Cake Extension for Visual Studio](/docs/integrations/editors/visualstudio/).  This release has been baking for quite a while now, the previous release shipped in 2017, but we are hoping that it will be worth the wait.
+
+The main focus of this release, was to start to bring the extension within feature parity of the [Cake Extension for VSCode](/docs/integrations/editors/visualstudio). This is still a long way off, but we have made some good progress in the journey.  That is the main reason why this is a 0.3.0 release, rather than a 1.0.0 release.
```

Comment:
Should link to VSCode:

```suggestion
The main focus of this release, was to start to bring the extension within feature parity of the [Cake Extension for VSCode](/docs/integrations/editors/vscode). This is still a long way off, but we have made some good progress in the journey. That is the main reason why this is a 0.3.0 release, rather than a 1.0.0 release.
```

Reply:
Doh!